examples/sifive-hifive_unleashed-microsemi.dts

```diff
@@ -0,0 +1,622 @@
+/dts-v1/;
+
+/ {
+	#address-cells = <2>;
+	#size-cells = <2>;
+	compatible = "sifive,fu540g", "sifive,fu500";
+	model = "sifive,hifive-unleashed-a00";
+
+	aliases {
+		serial0 = &L28;
+		serial1 = &L29;
+	};
+
+	chosen {
+	};
+
+	firmware {
+		sifive,fsbl = "YYYY-MM-DD";
+	};
+
+	L3: cpus {
```

Comment:
The labels are a bit meaningless (but still better than explicit phandles :)

Reply:
Ya, I agree.  They're auto-generated, though, and I'm afraid I won't have time to make something prettier.

Reply:
Ok, I see

Reply:
Why the local interrupt-controller (HLIC) doesn't have phandle & linux,phandle entry ?
Is it automatically generated at FSBL layer ?

Reply:
Who adds the phandles then ? Especially for interrupt-controller.
bbl expects a phandle in during query hart.